frontend/taipy/src/DataNodeChart.tsx

```diff
@@ -237,19 +237,24 @@ const DataNodeChart = (props: DataNodeChartProps) => {
 
     const [config, setConfig] = useState<ChartConfig | undefined>(undefined);
     useEffect(() => {
+        let localConf;
```

Comment:
No init value? Or relying on the language's default?

Reply:
> No init value? Or relying on the language's default?

Default is undefined and that's good with me